terraform/main.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
+  backend "s3" {
+    bucket                  = "aeternity-terraform-states"
+    key                     = "aeternitystate.tfstate"
+    region                  = "us-east-1"
+    shared_credentials_file = "/aws/credentials"
+    profile                 = "aeternity"
+  }
+}
+
+provider "aws" {
+  version                 = "1.24"
+  region                  = "ap-southeast-1"
+  alias                   = "ap-southeast-1"
+  shared_credentials_file = "/aws/credentials"
```

Comment:
let's use environment variables for credentials configuration, also the profile below.

Reply:
removed, leftover from local testing